js/components/MilkdropWindow/index.js

```diff
@@ -151,6 +156,16 @@ class PresetsLoader extends React.Component {
   }
 }
 
+async function loadInitialPreset(presetUrl) {
+  if (presetUrl) {
+    const response = await fetch(presetUrl);
+    const responseBody = await response.json();
```

Comment:
Maybe we could wrap these lines in a try/catch and show some alert if it fails?

Reply:
sounds good